src/pybind/mgr/dashboard/controllers/__init__.py

```diff
@@ -64,6 +64,11 @@ def __call__(self, cls):
         return cls
 
 
+class UiController(Controller):
+    def __init__(self, path):
+        super(UiController, self).__init__(path, base_url="/ui")
```

Comment:
What about using /api-ui or /ui-api to make it more clear for what this is for?

Reply:
I don't have a strong opinion on this, but I prefer the smaller string `"/ui"`.

Reply:
@rjfd My only argument here is that `/ui` can cause the illusion that this endpoint will return a HTML.

I will change it to `/api-ui` in the hope that it will make it clearer that this endpoint will return a JSON.

Reply:
That's exactly my counter-argument :-) by using `/ui-api` you might lose the flexibility of returning HTML in the future. But I'm not strongly against...

Reply:
@votdev decorator was renamed to `UiApiController`, and base_url is now `/ui-api`.